docs/references/errors.md

```diff
@@ -111,32 +112,32 @@ The provided subscription is invalid. Subscriptions should have:
 
 Please look at the [Kafka documentation page](https://docs.restate.dev/services/sdk/kafka) for more details on subscriptions and event handlers.
 
-## RT0001
+## RT0001 {#RT0001}
 
-The invocation response stream was aborted due to the timeout configured in `worker.invoker.response_abort_timeout`.
+The invocation response stream was aborted due to the timeout configured in `worker.invoker.abort_timeout`.
 This timeout is fired when Restate has an open invocation, and it's waiting only for response messages, but no message is seen for the configured time.
 
 Suggestions:
 
 * Check for bugs in your code. Most likely no message was sent to Restate because your code is blocked and/or reached a deadlock.
 * If your code is supposed to not send any message to Restate for longer than the configured timeout, because for example is doing a blocking operation that takes a long time, change the configuration accordingly.
 
-## RT0002
+## RT0002 {#RT0002}
 
 Cannot start Restate because the configuration cannot be parsed. Check the configuration file and the environment variables provided.
 
 For a complete list of configuration options, and a sample configuration, check https://docs.restate.dev/restate/configuration
 
-## RT0003
+## RT0003 {#RT0003}
 
-The invocation failed because the invoker received a message from a service endpoint larger than the `worker.invoker.message_size_limit`.
+The invocation failed because the invoker received a message from a service larger than the `worker.invoker.message_size_limit`.
```

Comment:
```suggestion
The invocation failed because Restate received a message from a service larger than the `worker.invoker.message_size_limit`.
```

Reply:
Also auto generated. Let's fix the error messages here: https://github.com/restatedev/restate/tree/main/crates/errors/src/error_codes